cirq/circuits/text_diagram_drawer.py

```diff
@@ -171,13 +190,26 @@ def render(self,
         for (x, y), v in self.entries.items():
             grid[y][x] = v
 
+        # Prepare Moment groups.
+        if self.groups_are_vertical:
```

Comment:
Maybe when defining the characters, we should define their transpose. That would allow these to be merged together and might be more generally useful.

Reply:
Oops. Obsoleted by my huge design change comment.